src/handlers.js

```diff
@@ -1,4 +1,47 @@
 // 1. handler to return the homepage('index.html and all resources from
-      //index.html')
+//index.html')
 // 2. handles search --> making a call to request.js and processes the result;
 // 3. handles other and returns 404 error.
+const path = require("path");
+const fs = require('fs');
+const apiRequest = require('./apiRequest');
+const router = require('./router.js');
+
+let handlers = module.exports = {}
+
+handlers.serveLanding = (request, response) => {
+  fs.readFile(path.join(__dirname, '..', 'public', 'index.html'), (error, file) => {
+    if (error) throw error;
+    response.writeHead(200, {
+      'content-type': 'text.html'
+    });
+    response.end(file);
+  });
+}
+
+handlers.servePublic = (request, response) => {
+  fs.readFile(path.join(__dirname, '..', 'request.url'), (error, file) => {
+    if (error) throw error;
+    let content = request.url.split('.')[1];
```

Comment:
I think that the request.url for, for example, main.css would be something like '/main.css' as in the index.html (would be good to check this), splitting on '.' will return 'css'. 'css' is not a property key on the contentType object, so doing contentType[content] would return undefined.